addons/payment_adyen/static/src/js/payment_form.js

```diff
@@ -173,6 +173,35 @@ odoo.define('payment_adyen.payment_form', require => {
                             showPaymentMethods: true,
                             showPayButton: false,
                             setStatusAutomatically: true,
+                            onSelect: (component) => {
+                                if (component.props.name === "PayPal") {
+                                     if (!this.paypalForm){
+                                         // create div element to render PayPal component
+                                         const el = document.createElement("div");
+                                         document.getElementById(
+                                             `o_adyen_dropin_container_${paymentOptionId}`
+                                         ).appendChild(el);
+                                         el.setAttribute("id", "o_adyen_paypal_component");
+                                         el.setAttribute("className", "d-none");
```

Comment:
Set and remove classes with `.classList.add/remove`, so you can work with multiple classes.
```suggestion
                                         el.classList.add("d-none");
```

Reply:
In this case, however, you don't need to set `d-none` because the element is being created and shown immediately after.